### DISCUSSION TITLE
Touchscreen Support

### DISCUSSION THREAD

**Discussion by jodave911** [valnoxy/t480-oc#40] [Q&A] [answered]
Hello Valnoxy, Can't believe we are both the same age. 
So, I followed your guide and successfully installed macOS Monterey on my T480. 
The only issue is that the touchscreen doesn't work.   

- Joshua D

**Comment by valnoxy** [comment, marked as answer]
Hey 👋,
I'm currently working on Touchscreen support. I already have a touchscreen version in the development branch, but it is currently only compatible with Ventura. In the next few days I will port it to the other versions as well.

**Reply by jodave911** [reply]
Got it.  Looking forward for the updates.
:)

- Joshua D
